Mathlib/Combinatorics/Composition.lean

```diff
@@ -232,9 +222,10 @@ theorem sizeUpTo_le (i : ℕ) : c.sizeUpTo i ≤ n := by
 #align composition.size_up_to_le Composition.sizeUpTo_le
 
 theorem sizeUpTo_succ {i : ℕ} (h : i < c.length) :
-    c.sizeUpTo (i + 1) = c.sizeUpTo i + c.blocks.nthLe i h := by
+    c.sizeUpTo (i + 1) = c.sizeUpTo i + c.blocks.get ⟨i, h⟩ := by
   simp only [sizeUpTo]
   rw [sum_take_succ _ _ h]
+  rfl
```

Comment:
This looks like a missing lemma too?

Reply:
This should resolve after adapting `sum_tage_succ`. 
I added a porting note.